code/game/gamemodes/zombie/zombie.dm

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
...Why did you comment out the entire gamemode? This entire code allows zombies to be mulligan.